gem/lib/rb_sys/extensiontask.rb

```diff
@@ -45,6 +45,7 @@ def init(name = nil, gem_spec = nil)
         gemspec.files.reject! { |f| f.end_with?(".rs") }
         gemspec.files.reject! { |f| f.match?(/Cargo.(toml|lock)$/) }
         gemspec.files.reject! { |f| extconf.end_with?(f) }
+        gemspec.files.reject! { |f| extconf.match?(/Cargo.(toml|lock)$/) }
```

Comment:
pretty sure `extconf.match?(/Cargo.(toml|lock)$/)` will never be true 😕

Reply:
Thanks for your contribution! I think what we want is:

```suggestion
        gemspec.extensions.reject! { |f| f.end_with?("Cargo.toml") }
```

This will prevent rubygems from attempting to compile the gem with `cargo`, which is unnecessary for precompiled gems. Does that make sense?

Reply:
Didn't consider `gemspec.extensions` 😲  Yeah makes sense 👍